kafka-handler/pom.xml

```diff
@@ -180,5 +197,28 @@
         </executions>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.avro</groupId>
+          <artifactId>avro-maven-plugin</artifactId>
+          <version>1.8.2</version>
```

Comment:
can you please use the version of the top level project ?

Reply:
Note: Confluent uses `1.8.1`

Link: https://github.com/confluentinc/common/pull/130

Reply:
Changed to latest version.